src/iqstat.cpp

```diff
@@ -357,3 +227,116 @@ void usage() {
     }
     printReleaseInfo( "iqstat" );
 }
+
+auto show_RuleExec( char *userName,
+                    const char *ruleName,
+                    int allFlag,
+                    bool brief ) -> int
```

Comment:
This function always returns 0. Not sure what it means, but just pointing that out as we consider the return type for this

Reply:
Yep. There were two functions originally, showRuleExec and showRuleExecBrief.  They both always returned 0.  This function is the synthesis of those two.

Reply:
Okay. Looking at this again, this is sort of related to my comment further down about error handling. Feel free to resolve this in favor of the other.

Reply:
ok. closing in favor of other solution, handled below.